recipes/rmm/meta.yaml

```
@@ -0,0 +1,60 @@
+{% set name = "rmm" %}
+{% set version = "21.12.00" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/rapidsai/rmm/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 71802422a04f2e0a828f7b7ec77e5cd1d24361d6dc5bf60a534659e4c2e08a44
+  folder: rmm
+
+build:
+  number: 0
+  skip: true  # [(not linux) or (cuda_compiler_version !=  "11.2") or (aarch64) or (py < 37) or (py >= 39)]
+
+  script_env:
+    - INSTALL_PREFIX=${PREFIX}
+    - CUDA_HOME  # [ppc64le]
+
+  script:
+    - cd rmm/python
+    - {{ PYTHON }} -m pip install . --no-deps -vv
+
+  missing_dso_whitelist:
+    - '*/libcuda.*'  # [linux]
+
```

```suggestion
  ignore_run_exports:
    - spdlog
```
From the build logs, it looks like spdlog (which is advertised as header-only) is not needed in the run environment. It also looks like librmm is header-only because the librmm package does not contain any shared libraries, but maybe it has run exports because of JIT compiling? @leofang, do you know why librmm has run_exports when it is header-only?

```
WARNING (rmm): run-exports library package conda-forge::spdlog-1.9.2-h4bd325d_1 in requirements/run but it is not used (i.e. it is overdepending or perhaps statically linked? If that is what you want then add it to `build/ignore_run_exports`)
WARNING (rmm): run-exports library package conda-forge::librmm-21.12.00-hbecc9b5_1 in requirements/run but it is not used (i.e. it is overdepending or perhaps statically linked? If that is what you want then add it to `build/ignore_run_exports`)
```

---

@conda-forge/core can confirm (especially @kkraus14), IIUC I think there's a growing consensus that even header-only libraries should do a `run_exports` to avoid subtle symbol leak issues. You can check that `spdlog`, which is also a header-only library, also has it.

---

but for this case `rmm` can't be a build time dependency for other projects (I think), so probably it's fine to ignore both `spdlog` and `librmm` as you suggested